v2/charts/azure-service-operator/values.yaml

```diff
@@ -189,12 +189,17 @@ revisionHistoryLimit: 10
 
 # Specify security settings for a Container
 # Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
-# It is not recommended to reduce the restrictions in this list, but additional restrictions outside of the default set 
+# It is not recommended to reduce the restrictions in this list, but additional restrictions outside the default set
 # can be applied. If you believe additional securityContext configuration should be specified by default
 # please raise an issue.
 securityContext:
+  runAsUser: 65536
+  runAsNonRoot: true
```

Comment:
Will you be adding a runAsGroup here as well? Seems to be the same value:

![image](https://github.com/user-attachments/assets/f4d50d2b-9837-4147-aad5-a77eba5c9f62)

Reply:
See here: https://github.com/GoogleContainerTools/distroless/blob/main/common/variables.bzl#L18

Reply:
yes, will add runAsGroup as well, and good callout on the ID. I thought it was any ID which is nonroot which is technically true, but actually 65532 is the designated nonroot user for distroless images so we should be using that.

We already _are_ in the sense we set USER nonroot:nonroot in our dockerfile, but we should use the same values in the Kuberentes configuration too.